oat-sa/tao-core
# Feature/tao 8011/icons for templates

https://oat-sa.atlassian.net/browse/TAO-8011

This is just an early PR to get 2 new icons into the TAO icon font, before starting work on the Item Templates task linked above.

To test:
1. check out the branch
2. install `taoDevTools` extension
3. in the backoffice, navigate to Tools > TAO Icon Font
4. you should see 200-something icons including `template` and `template-from`
